### PR TITLE
Highlight matched strings in the log lines with the include option

### PR DESCRIPTION
**Pull request by superbrothers**
This PR changes to highlight matched strings in the log lines with the include option.

image

Closes https://github.com/stern/stern/issues/195

### CODE REVIEW

File: stern/tail.go [hunk 1]
```diff
@@ -99,6 +99,25 @@ func (o TailOptions) IsInclude(msg string) bool {
...
+
+	for _, rin := range o.Include {
+		msg = rin.ReplaceAllStringFunc(msg, func(part string) string {
+			matched = true
+			return colorHighlight(part)
+		})
+
+		if matched {
+			break
+		}
```
Comment (tksm) [inline stern/tail.go:115]: We can specify the include option multiple times, but it seems that this logic highlights first matching regular expression only. Was this done intentionally?

image

Comment (superbrothers) [reply stern/tail.go:115]: Oh, I thought I should only highlight the first match, but I was wrong. I will fix it.

Comment (superbrothers) [reply stern/tail.go]: I've fixed it in 06904a6.

image

File: stern/tail.go [hunk 2]
```diff
@@ -99,6 +103,30 @@ func (o TailOptions) IsInclude(msg string) bool {
 	return false
 }
 
+var colorHighlight = color.New(color.FgRed, color.Bold).SprintFunc()
+
+func (o TailOptions) HighlightMatchedString(msg string) string {
+	if o.reHightlight == nil {
```
Comment (tksm) [inline stern/tail.go:109]:
```suggestion
	if len(o.Include) == 0 {
		return msg
	}
	
	if o.reHightlight == nil {
```

If the include option is not specified, the following process is unnecessary, so I prefer to add an early return.

**Comment by superbrothers** [comment]
I've squashed this PR.

**Review by tksm — APPROVED** [review]
LGTM 🎉

**Comment by tksm** [comment]
📝 I like that `-i ^` allows us to highlight lines without filtering.

image

**Pull request merged**